package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "auth0-deploy-extensions",
-  "version": "3.2.1",
```

Comment:
Patch bumps automatically go out to all customers with the extensions installed.

A bump to a dependency that is excluded (see below in `externals`) means that it needs to be in webtask. Due to this requirement, when we bump an excluded dependency, we do a minor bump here.

This means that the extension will not break customers who are using a version of webtask that doesn't contain the updated package.

Reply:
Bumped to 3.3.0 in  64b2a2a